packages/lab/src/__tests__/__e2e__/overlay/Overlay.cy.tsx

```diff
@@ -3,55 +3,113 @@ import * as overlayStories from "@stories/overlay/overlay.stories";
 import { checkAccessibility } from "../../../../../../cypress/tests/checkAccessibility";
 
 const composedStories = composeStories(overlayStories);
-const { OverlayTop, OverlayRight, OverlayBottom, OverlayLeft } =
-  composedStories;
+const { Default, OverlayRight, OverlayBottom, OverlayLeft } = composedStories;
 
 describe("GIVEN an Overlay", () => {
   checkAccessibility(composedStories);
 
+  describe("WHEN rendered", () => {
+    it("THEN it should show Overlay on trigger element press", () => {
+      cy.mount(<Default />);
+
+      cy.realPress("Tab").realPress("Enter");
```

Comment:
you could unchain there to improve readability 

```suggestion
      cy.realPress("Tab");
      cy.realPress("Enter");
```